magento/pwa-studio
# [PWA-835] Localize Checkout

## Description

Localize all strings within the Checkout area of the application. Include localized strings in default en_US language pack

## Related Issue





https://jira.corp.magento.com/browse/PWA-835

## Acceptance 



### Verification Stakeholders

@sirugh 
@dpatil-magento 
### Specification



### Verification Steps


1. Add a product to cart
2. Go to Checkout page
3. Go through each step of the checkout process (with guest and with authorized user)
3. Verify strings are translated where applicable

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
